.github/workflows/pg-17-migtests.yml

```diff
@@ -69,17 +69,18 @@ jobs:
         run: |
           cd installer_scripts
           yes | ./install-yb-voyager --install-from-local-source --only-pg-support
+          sudo apt-get -y install postgresql-17
```

Comment:
should we start shipping PG 17 via installer script now?
Because 
If we are gonna detect till PG17, then installing older version during setup is of no use.
It will be a manual step for the user unnecessarily.

Lets discuss this in the standup tomorrow.

Reply:
Yes that is already in plan. I believe there is already ticket for that https://yugabyte.atlassian.net/browse/DB-14593

Reply:
cool, just add a TODO here saying that we need to remove this once installer script picks up 17